src/openloco/scenario.cpp

```diff
@@ -23,4 +66,97 @@ namespace openloco::scenario
     {
         call(0x0043C90C);
     }
+
+    // 0x0046E239
+    // NB: called in sub_46E2C0 below, as well in openloco::date_tick.
+    void sub_46E239()
+    {
+        for (uint8_t i = 0; i < 32; i++)
```

Comment:
I think the entire highlighted area is useless, as `_525EDE` is only ever filled with zeroes.
<img width="420" alt="loc_46e238" src="https://user-images.githubusercontent.com/737603/59449489-7a57cb00-8e07-11e9-98c2-dd31bb5aac8d.png">

Reply:
What makes you sure it's zeros? I think it represents the highest 16 bits of a `money48` struct.

Reply:
As I said, `525EDE` is only ever filled with zeroes. Just look at the references.

Reply:
You can see what CS tried to do but he performed the calc wrong. The lower bits from the second shrd were meant to be put into 525EDE but instead are discarded. So 525EDE does indeed do nothing.